flipperdevices/flipperzero-firmware
# RFID: add support for Kantech IOProx cards

# What's new

- Adds supporting for reading, writing, and emulating Kantech IoProx 125kHz RFID cards; as commonly used in e.g. building access control systems.  

# Verification 

If you have an IOProx card:

- Use _Read_ from the _125kHz RFID_ menu, and read an IoProx tag.
- Use `rfid read` from the CLI, and read an IoProx tag.

If you have a T55xx card:

- Use _Add Manually_ from the _125kHz RFID_ menu, and enter in a valid code, such as `3B01AEB5`.
- Select _Emulate_, and validate that the emulated tag is readable from e.g. another flipper.
- Select _Write_, and write to a T55xx card.
- Use _Read_ from the _125kHz RFID_ menu, and read the card.

Repeat with the CLI:

- Use `rfid emulate IoProx 3B01AEB5` and validate that the emulated tag is readable from e.g. another flipper.
- Use `rfid write IoProx 3B01AEB5` to write to a T55xx card.
- Use `rfid read` to read from the card, and verify that it reads as an IoProx card with the given code.


# Checklist (For Reviewer)
